[#98] Space character dropped at line break when converting to PDF (#192)
se-edu/addressbook-level3: docs/UserGuide.md

* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
  e.g. if the command specifies `help 123`, it will be interpreted as `help`.

> * If you are using a PDF version of this document, be careful when copy-pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.

Reviewer: Imo, is quite odd to put this under the **"notes about the command format"** section as it does not fit well with the other pointers in it. Perhaps, it will be better to be put it in a separate section like the FAQ or below the features header outside of the **"notes about the command format"** section?

Reviewer 2: In my opinion, positioning this information here is beneficial as it pertains directly to the use of commands. From personal experience, I don't typically read through the entire User Guide before starting to use the commands. Therefore, having notes about potential pitfalls or issues within the command format section can be extremely useful.

Furthermore, I view the FAQ as an additional, somewhat less critical feature, which should not interfere with or impede the fundamental understanding and usage of the commands. As such, I believe it's acceptable to prioritize the 'Note about the command' section over the FAQ.

Reviewer 3: Thanks for inputs @jundatan @SPWwj 
Yes it is a bit odd to have in under 'notes about the command format'. Just below it as a separate box is one viable option. Putting it in the FAQ is not a good idea, as @SPWwj mentioned. For now, let's keep it in the current place as it is less disruptive to the flow of the document.

> * If you are using a PDF version of this document, be careful when copy-pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.

Reviewer 2: A minor alteration suggestion:
It may be less confusing to users if we adjust the wording regarding the copy-pasting commands. Instead of suggesting that AB3 has a copy-paste command, we could articulate it as 'copying and pasting the command from the PDF version of the User Guide.'

> * Be careful when copy-pasting commands that cut across a single line from this user guide as `Sample commands` that span more than a single line may drop a space at the line-break when copied over to the application. 

Reviewer 4: Can consider shortening `Sample commands ...` to `these commands` to make it less verbose

```suggestion
* Be careful when copy-pasting commands that span more than a single line from this user guide as the space character at the line-break may be lost when these commands are copied over to the application. 
```

> * Be careful when copy-pasting commands that cut across a single line from this user guide as `Sample commands` that span more than a single line may drop a space at the line-break when copied over to the application. 

Reviewer 3: Another alternative that mentions PDF files specifically.
```suggestion
* If you are using a PDF version of this document, be careful when copy-pasting commands that span multiple lines, as spaces surrounding line breaks can get omitted when copy-pasted from a PDF file. 
```
